go/test/endtoend/onlineddl/vrepl_suite/testdata/update-pk-col-uppercase/alter

```diff
@@ -0,0 +1 @@
+engine=innodb
```

Comment:
So is this alter which is triggering that bug. I didn't see the case of the pk being changed explicitly in this test, as I had expected.